content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/processors/oembed.html

```diff
@@ -0,0 +1,22 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-template.render="${@ options}">
+    <img data-sly-test="${options.response.type == 'photo'}"
+         src="${options.response.url}" width="${options.response.width}" height="${options.response.height}" alt="${options.response.title}" title="${options.response.title}"/>
+    <a data-sly-test="${options.response.type == 'link'}"
+       href="${options.response.url}">${options.response.title}</a>
+    <sly data-sly-test="${options.response.type == 'video' || options.response.type == 'rich'}">${options.response.html @ context = 'unsafe'}</sly>
```

Comment:
If the context has to be unsafe, then the decision must be made based on the provider/source rather than the response type to ensure that only trusted sources are consumed in such unsafe context.

Reply:
@acollign - addressed in 0c8579cdc14

cc @vladbailescu, @bpauli, @jckautzmann